projects/speech_commands/experiments.cfg

```diff
@@ -77,23 +77,51 @@ model_type = "linear"
 ;batch_size = 32
 ;model_type = "linear"
 ;use_batch_norm = True
-
 ;
+
 ; Gets about 96.6% test error after 15 iterations. More iters might help.
-;
 ;[resnet9]
-;iterations = 15
+;iterations = 20
+;learning_rate_factor = [0.8,0.9]
+;learning_rate = [0.04, 0.05]
+;momentum = 0.9
+;weight_sparsity = 1.0
+;dropout = [0.0, 0.5]
+;log_interval = 40
+;test_noise_every_epoch = False
+;batches_in_epoch = 214
+;batch_size = 96
+;model_type = "resnet9"
+
+; Best resnet - multiple seeds
+;[resnet9Seeds]
+;iterations = 20
 ;learning_rate_factor = 0.8
 ;learning_rate = 0.04
 ;momentum = 0.9
 ;weight_sparsity = 1.0
-;dropout = 0.0
+;dropout = 0
 ;log_interval = 40
 ;test_noise_every_epoch = False
 ;batches_in_epoch = 214
 ;batch_size = 96
 ;model_type = "resnet9"
+;seed = [43, 44, 45, 46, 47, 48, 49, 50]
 
+; Best resnet - two more seeds
+[resnet9TwoMoreSeeds]
+iterations = 20
+learning_rate_factor = 0.8
+learning_rate = 0.04
+momentum = 0.9
+weight_sparsity = 1.0
+dropout = 0
+log_interval = 40
+test_noise_every_epoch = False
+batches_in_epoch = 214
+batch_size = 96
+model_type = "resnet9"
+seed = [51, 52]
```

Comment:
This is not necessary. The seed is updated on the reset based on the repetition.
https://github.com/numenta/htmresearch/blob/master/htmresearch/frameworks/pytorch/mnist_sparse_experiment.py#L65

Reply:
Good point. I'll fix these after Wed 😄